src/utils/authUtils.ts

```diff
@@ -16,6 +17,23 @@ export const setAuthErrorMessage = (
   errorMessage: string,
 ) => dispatch({ type: 'SET_ERROR_MESSAGE', errorMessage });
 
+export const forgotPassword = async (
+  dispatch: AuthDispatch,
+  params: { email: string },
+) => {
+  const auth = getAuth();
+  await sendPasswordResetEmail(auth, params.email)
+    .then(() => {
+      // eslint-disable-next-line no-console
+      console.log('Email Sent!');
```

Comment:
let's get rid of the console log and add a success message when the email is successfully sent!

Reply:
removed